core/spec/models/spree/calculator/refunds/default_refund_amount_spec.rb

```diff
@@ -36,7 +37,7 @@
         order.adjustments.first.update_attributes(amount: adjustment_amount)
       end
 
-      it { is_expected.to eq line_item_price - (adjustment_amount.abs / line_item_quantity) }
+      it { is_expected.to eq line_item_price - (adjustment_amount.abs * ( line_item_price / line_item_total )) }
```

Comment:
Can you replace this call with the actual number its supposed to be, please. Doing it on the computed amount from the setup makes it tougher to figure out what should be happening (though this calculation would be nice to keep in a comment or something).

You can take a look at the tax specs and see how much more readability it adds when you're really trying to figure out whats happening: https://github.com/solidusio/solidus/blob/master/core/spec/models/spree/calculator/default_tax_spec.rb